package.json

```diff
@@ -11,9 +11,10 @@
 		"lint": "tslint '*/*/src/**/*.ts' '*/*/test/**/*.ts'",
 		"build": "lerna run build",
 		"watch": "lerna run watch",
-		"prep-release": "./scripts/prepareRelease.sh",
-		"validate-release": "./scripts/validateRelease.sh",
-		"publishCanary": "lerna publish --canary",
+		"release:prep": "./scripts/prepareRelease.sh",
+    "release:validate": "./scripts/validateRelease.sh",
+    "updateXml": "lerna exec -- \\$LERNA_ROOT_PATH/scripts/updatePluginXmlVersion.sh",
```

Comment:
I don't know if we should have this as a npm script, since it is not meant to be run standalone, or is it?

Reply:
The current approach wouldn't make any changes if ran by itself on updated files, we can remove it from package.json and use lerna exec directly from the build scripts but I personally see no harm